chia/full_node/mempool_manager.py

```diff
@@ -667,22 +667,16 @@ async def new_peak(
         self.mempool.fee_estimator.new_block(FeeBlockInfo(new_peak.height, included_items))
         return txs_added
 
-    async def get_items_not_in_filter(self, mempool_filter: PyBIP158, limit: int = 100) -> List[MempoolItem]:
-        items: List[MempoolItem] = []
+    def get_items_not_in_filter(self, mempool_filter: PyBIP158, limit: int = 100) -> List[SpendBundle]:
```

Comment:
given that you could make this change without updating a test suggest we don't have any tests for this function. Do you know if any test exercises this functionality?

Reply:
I found none so I sent https://github.com/Chia-Network/chia-blockchain/pull/14416